sources/packages/api/src/route-controllers/institution/institution.controller.service.ts

```diff
@@ -67,4 +71,17 @@ export class InstitutionControllerService {
       hasBusinessGuid: !!institutionDetail.businessGuid,
     };
   }
+
+  /**
+   * Get the list os all institutions types to be returned in an option
+   * list (key/value pair) schema.
+   * @returns institutions types in an option list (key/value pair) schema.
+   */
+  async getInstitutionTypeOptions(): Promise<OptionItemAPIOutDTO[]> {
+    const institutionTypes = await this.institutionTypeService.getAll();
```

Comment:
i know not part of the PR, but we can rename the function `getAll`

Reply:
Rename to what exactly?

Reply:
I believe its fetching institution type. so `getInstitutionTypes` or `getAllInstitutionTypes`

Reply:
Renamed!